libs/platform/src/lib/form/multi-combobox/commons/base-multi-combobox.ts

```diff
@@ -714,7 +725,11 @@ export abstract class BaseMultiCombobox extends CollectionBaseInput implements A
         // initial data fetch
         const map = new Map();
         map.set('query', '*');
-        map.set('limit', 12);
+
+        if (!this.limitless) {
+            map.set('limit', MAP_LIMIT);
```

Comment:
Is there any particular reason behind that "12"? or it can be anything user wants and can be put out to configuration?

Reply:
It was implemented before I touch this component, I just extracted it to constant